Skip wait if script download fails

millerdev opened the pull request:
### Rollback instructions

<!--
If this PR follows standards of revertability, check the box below.
Otherwise replace it with detailed instructions or reasons a rollback is impossible.
-->

- [x] This PR can be reverted after deploy with no further considerations

millerdev committed:
Skip wait if script download fails

snopoke commented on docker/hq-compose.yml:
```diff
@@ -124,7 +124,12 @@ services:
+          CUSTOM_INIT_SCRIPT: |
+            set -e
+            URL="https://raw.githubusercontent.com/vishnubob/wait-for-it/c096cface5fbd9f2d6b037391dfecae6fde1362e/wait-for-it.sh"
+            curl $URL > wait.sh || exit 0  # skip wait if script download fails
```
I think this might need a timeout, in the kafka logs I looked at the container was still waiting on the the download.

millerdev committed:
Do not exit kafaka-startup.sh if download fails

Misunderstood how this worked. See
https://github.com/wurstmeister/kafka-docker/blob/master/start-kafka.sh#L146

millerdev committed:
Fix conditional and add curl timeout
